strings/brain_damage_lines.json

```diff
@@ -61,7 +61,8 @@
         "@pick(semicolon)N-NYAAAAAA~",
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
-        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)"
+        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
+		"@pick(semicolon)okay i'm reporting shitcurity don't grab prisoners with you is 1 shitcurity is 2 and asscurity is 3 = permaban i'm reporting"
```

Comment:
there's a whole setup for randomly picking what type of "is it a bug" to pick and brain damage stuff is the ramblings of the insane and damaged